WebDriverAgentLib/Utilities/FBImageUtils.m

```diff
@@ -45,61 +45,3 @@ BOOL FBIsPngImage(NSData *imageData)
   UIImage *image = [UIImage imageWithData:imageData];
   return nil == image ? nil : (NSData *)UIImagePNGRepresentation(image);
 }
-
-#if TARGET_OS_TV
-NSData *FBAdjustScreenshotOrientationForApplication(NSData *screenshotData)
-{
-  return FBToPngData(screenshotData);
-}
-#else
-NSData *FBAdjustScreenshotOrientationForApplication(NSData *screenshotData, UIInterfaceOrientation orientation)
-{
-  if (nil == screenshotData) {
-    return nil;
-  }
-
-  UIImageOrientation imageOrientation;
-  if (FBConfiguration.screenshotOrientation == UIInterfaceOrientationUnknown) {
-    if (SYSTEM_VERSION_LESS_THAN(@"11.0")) {
-      // In iOS < 11.0 screenshots are already adjusted properly
-      imageOrientation = UIImageOrientationUp;
-    } else if (orientation == UIInterfaceOrientationLandscapeRight) {
-      imageOrientation = UIImageOrientationLeft;
-    } else if (orientation == UIInterfaceOrientationLandscapeLeft) {
-      imageOrientation = UIImageOrientationRight;
-    } else if (orientation == UIInterfaceOrientationPortraitUpsideDown) {
-      imageOrientation = UIImageOrientationDown;
-    } else {
-      return FBToPngData(screenshotData);
-    }
-  } else {
-    switch (FBConfiguration.screenshotOrientation) {
```

Comment:
Can we keep this settings in https://github.com/appium/WebDriverAgent/pull/482/files#diff-50f20c000d3b1215568d23ae0d9e54468c4aa940353be843759688cd69571de2R129 to set the screenshot orientation forcefully?
This setting's necessity happens on real devices and iPad devices/simulators. (Especially landscape mode)

Reply:
I tested the current implementation with different device orientations and it seems to be fine now for all of them. Could you please tell me which combination of device/orientation does not work there?

Reply:
I don't have the device, but the original report I introduced this was iOS 13.0 or around and iPhone 8(?) with a gaming app. A customer told me some iPad on their env as well. (But I don't remember this iPads' concrete versions)

btw, this case also I do not have a device, but previously I observed Xcode 10.3 had this wrong coordinate issue but Xcode 11.4 or around did not have it. (I forgot this exact environment...)
So, potentially newer Xcode versions fixed these issues.

Reply:
added some code to enforce image orientation if requested